### ISSUE TITLE
Next Step Microgrant: SuperFun.social - A gamified Social Fun and Entertainment Dapp.

### ISSUE THREAD

**Issue by jaydippatel83** [filecoin-project/devgrants#1151]
### 1. What is your project? (max 100 words)
​​A gamified social Fun and Entertainment Dapp built using Filecoin - IPFS and Lens protocol opensource social graph that allows users to own the content they create, participate in online contests and play games.  

### 2. Project links
  
Link to GitHub repo: https://github.com/jaydippatel83/superfun_lens_demo
Link to demo or website:  www.Superfun.social
License: MIT license. 

### 3. How is IPFS, Filecoin, or related technology used in this project? (max 100 words)

From social profiles to NFT posts, NFT metadata will get stored on FIlecoin - IPFS. Here creator has the power to earn by charging fees to view the content (NFT Posts) they create and sell in the marketplace. 

Here users can own Social Graph and Control the data by using filecoin as a storage layer and Lens Protocol as an open-source social graph. 

SuperFun Store:  will be a marketplace where creators will be able to sell PFPs/Avatar for games and social media profiles. Here all NFT metadata of the marketplace will get stored on NFT.Storage

### 4. How will you improve your project with this grant? What steps will you take to meet this objective? (max 200 words)

This grant will be utilized for:
The current version of SuperFun.Social includes follow, Posts, and Mirror part now we will integrate the full social graph of Lens with the Collect module to charge fees on posts, Report copyrighted or pornographic content 
Add support for Audio and Video NFT posts and store it on IPFS using infura.
Build modules for online contests where Web3 projects can create contests like meme competitions and creators can earn value for their creative content 
Create a mobile app prototype for SuperFun.Social using react native or Flutter. 

### 5. Do you agree to share grant reports upon request, including a final grant report at the end of the three-month period?
Yes for sure. 
  
### 6. Does your proposal comply with our Community Code of Conduct?
Yes

### 7. Links and submissions
Ethglobal: Ethonline Hackathon
https://ethglobal.com/showcase/superfun-social-mg7da

* How did you learn about our microgrant program?
Got an email from Papilon and Niki from Filecoin Team after the hackathon. 
  
* For each team member(s), please list name, email, GitHub account, and role in the project.
Jaydip patel - Blockchain Developer
 GitHub: https://github.com/jaydippatel83
 Email: jaydippatel83.pj@gmail.com

Dhruv Sathwara  - Frontend Developer 
Github:https://github.com/DhruvSathavara
Email: dhruvsathavara7228@gmail.com

**Comment by jaydippatel83** [comment]
Development update: Homepage UI has been updated - Report post module has been integrated

**Comment by ErinOCon** [comment]
This microgrant has been approved! We will send an email to confirm payment details. 

Please note that your final grant report is due within three months and can be submitted [here](https://airtable.com/shrrtSlgdVvrPmAvr). Please let us know if you have any questions!

**Comment by DhruvSathavara** [comment]
Recent developments: First, we've introduced a new feature: a Marketplace for gaming assets. This allows users to buy and sell limited edition gaming assets from a wide range of other users.

In addition, we've integrated a Chat-Bot based on ChatGPT. This tool is designed to assist users in writing better, more efficient content.

Next we are analysing Web3 gaming space to build ecosystem of Web3 games which will onboard individual developers, agencies, players.
